code/segmentedtickerplant/stplog.q

```diff
@@ -215,14 +228,21 @@ $[.sctp.chainedtp;
 // send end of day to subscribers, close out current logs, roll the day, 
 // create new and directory for the next day
 $[.sctp.chainedtp;
-  endofday:{[x;y]
+  endofday:{[date;data]
     .lg.o[`endofperiod;"flushing remaining data to subscribers and clearing tables"];
     .stpps.pubclear[.stplg.t];
-    p:.z.p+.eodtime.dailyadj;
+    p:.z.p;
     .lg.o[`endofday;"executing end of day for ",.Q.s1 .eodtime.d];
-    .stpps.end[x;y];
+    .stpps.end[date;data];
     if[p>.eodtime.nextroll:.eodtime.getroll[p];system"t 0";'"next roll is in the past"];
-    if[value `..createlogs; closelog each logtabs]
+    if[value `..createlogs; 
```

Comment:
same comment applies here as above - needs some comments + can we create a function out of it if it used more than once 
(I don't know if it is used more than once)